Pod/Classes/LocationManager.swift

```diff
@@ -10,244 +10,278 @@
+                    locationManager.requestWhenInUseAuthorization();
                 }
-            }else{
-                rejection(LocationManagerAuthorizationError.KeyInPlistMissing)
+
+            } else {
+
+                rejection(LocationManagerAuthorizationError.keyInPlistMissing)
                 rejected = true
             }
         }
+
         if !rejected {
-            self.askForLocationServicesFulfillments.append(fulfillment)
+            askForLocationServicesFulfillments.append(fulfillment)
         }
     }
     
     func startUpdatingLocationIfNeeded() {
-        if self.locationRequests.count > 0 || self.locationObservers.count > 0 {
-            self.askForLocationServicesIfNeeded()
-            self.locationManager.startUpdatingLocation()
+
+        if locationRequests.count > 0 || locationObservers.count > 0 {
+
+            askForLocationServicesIfNeeded()
+            locationManager.startUpdatingLocation()
         }
-        self.updateLocationManagerSettings()
+
+        updateLocationManagerSettings()
     }
     
     func stopUpdatingLocationIfPossible() {
-        if self.locationRequests.count == 0 && self.locationObservers.count == 0 {
-            self.locationManager.stopUpdatingLocation()
+
+        if locationRequests.count == 0 && locationObservers.count == 0 {
+            locationManager.stopUpdatingLocation()
         }
-        self.updateLocationManagerSettings()
+
+        updateLocationManagerSettings()
     }
     
     func updateLocationManagerSettings() {
-        let requestsDesiredAccuracy = self.locationRequests.map { (request) -> CLLocationAccuracy in
+
+        let requestsDesiredAccuracy = locationRequests.map { (request) -> CLLocationAccuracy in
             return request.desiredAccuracy ?? 0
-            }.minElement() ?? 0
+            }.min() ?? 0
         
-        let observersDesiredAccuracy = self.locationObservers.map { (observer) -> CLLocationAccuracy in
+        let observersDesiredAccuracy = locationObservers.map { (observer) -> CLLocationAccuracy in
             return observer.desiredAccuracy ?? 0
-            }.minElement() ?? 0
+            }.min() ?? 0
         
         let desiredAccuracy = min(requestsDesiredAccuracy,observersDesiredAccuracy)
-        if self.locationManager.desiredAccuracy != desiredAccuracy {
-           self.locationManager.desiredAccuracy = desiredAccuracy
+        if locationManager.desiredAccuracy != desiredAccuracy {
+           locationManager.desiredAccuracy = desiredAccuracy
         }
         
-        if self.locationRequests.count == 0 {
-            let observersDistanceFilter = self.locationObservers.map { (observer) -> CLLocationAccuracy in
+        if locationRequests.count == 0 {
+
+            let observersDistanceFilter = locationObservers.map { (observer) -> CLLocationAccuracy in
                 return observer.distanceFilter ?? 0
-                }.minElement() ?? 0
+                }.min() ?? 0
```

Comment:
Wrong padding int  (-1 ... previous version was incorrect too - even in previous `.min()...` call)